<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotUltrasound</class>
 <widget class="qMRMLWidget" name="RobotUltrasound">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>555</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="CollapsibleButton">
     <property name="text">
      <string>Control</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="connectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="connectionStatusLabel">
        <property name="text">
         <string>Status: Unknown.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="disconnectButton">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="homeButton">
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="relaxButton">
        <property name="text">
         <string>Relax</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Scan</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Center:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="setCenterManuallyButton">
        <property name="text">
         <string>Move arm manually</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="setAsCenterButton">
        <property name="text">
         <string>Use current position</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Go to:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QPushButton" name="centerButton">
        <property name="text">
         <string>Center</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QPushButton" name="endButton">
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QPushButton" name="landButton">
        <property name="text">
         <string>Down</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QPushButton" name="flyButton">
        <property name="text">
         <string>Up</string>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="3">
       <widget class="ctkSliderWidget" name="speedSlider">
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>50.000000000000000</double>
        </property>
        <property name="value">
         <double>5.000000000000000</double>
        </property>
        <property name="suffix">
         <string>%</string>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>speed</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="3">
       <widget class="ctkSliderWidget" name="angleRangeSlider">
        <property name="toolTip">
         <string>Set threshold value for computing the output image. Voxels that have intensities lower than this value will set to zero.</string>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="singleStep">
         <double>5.000000000000000</double>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>30.000000000000000</double>
        </property>
        <property name="value">
         <double>15.000000000000000</double>
        </property>
        <property name="suffix">
         <string> deg</string>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>angleRange</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="resetCenterToDefaultButton">
        <property name="text">
         <string>Reset to default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Volume reconstruction</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="2">
       <widget class="QPushButton" name="startVolumeReconstructionButton">
        <property name="text">
         <string>Scan</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="stopVolumeReconstructionButton">
        <property name="text">
         <string>Finish</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="resetVolumeReconstructionButton">
        <property name="text">
         <string>Initialize</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="liveUpdatesCheckBox">
        <property name="text">
         <string>Live updates</string>
        </property>
        <property name="SlicerParameterName" stdset="0">
         <string>liveUpdates</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header>ctkSliderWidget.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
